packages/libs/eda/src/lib/map/analysis/MarkerConfiguration/MapTypeConfigurationMenu.tsx

```diff
@@ -0,0 +1,55 @@
+import { useState } from 'react';
+import { H5 } from '@veupathdb/coreui';
+import { MarkerConfiguration } from '../appState';
+import TabbedDisplay, {
+  TabbedDisplayProps,
+} from '@veupathdb/coreui/dist/components/grids/TabbedDisplay';
+
+export interface MarkerConfigurationOption {
+  displayName: string;
+  icon?: JSX.Element;
+  renderConfigurationMenu: JSX.Element;
```

Comment:
Rename this `configurationMenu`. Reserve the `render` prefix for render functions.